ledger/storage/blob/blobmem.go

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This file doesn't have to begin with "blob". Let's call it just "memory" or "mem".

Reply:
done